.changeset/few-crabs-guess.md

```diff
@@ -0,0 +1,5 @@
+---
+"saleor-dashboard": patch
+---
+
+You can now see all options in the autocomplete select field
```

Comment:
- previously i couldn't see all?
- in the autocomplete select field - all autocompletes? which ones?